src/js/store.js

```diff
@@ -137,6 +138,18 @@ export const presetStores = [
   isChatInverted
 ];
 
+// -=- Language Migration -=-
+(async () => {
+  const language = SS('language', 'English');
+
+  await language.loaded;
+  await languages.loaded;
+
+  if (!languages.get().length <= 0) {
```

Comment:
This will only run when `languages` is not empty which is the opposite of what you wanted. Also, I think that a separate store which stores whether the migration happened would work better as the user may want to get no translations from YouTube chat.